.github/workflows/deploy-infra.yaml

```diff
@@ -40,6 +40,13 @@ jobs:
             ERROR=true
           fi
 
+          if [ ! -z "$ERROR" ]
+          then
+            echo
+            echo Make sure you the necessary secrets are defined in the respository.
+            false
+          fi
```

Comment:
This is still required to make the job fail in this step if the secret is not set. Otherwise, the job will fail later on.

I've improved the comment. I think it's sufficient. Do I need to explain why the secrets are necessary? (e.g. to access AWS and deploy the repo?)